flyway-core/src/main/java/org/flywaydb/core/internal/database/DatabaseTypeRegister.java

```diff
@@ -56,6 +57,7 @@ private static void registerDatabaseTypes() {
             for (DatabaseType dt : loader) {
```

Comment:
What is this loader. Can't we get added in the registeredDatabaseTypes in the loop only like others?

Reply:
Right, we can do that by adding the class name as DatabaseType service under `resources`. Will do the changes.